RapidWebView/src/main/java/in/workindia/rapidwebview/utils/RapidPermissionHelper.kt

```diff
@@ -0,0 +1,113 @@
+package `in`.workindia.rapidwebview.utils
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.app.DownloadManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Context.RECEIVER_EXPORTED
+import android.content.Context.RECEIVER_NOT_EXPORTED
+import android.content.IntentFilter
+import android.os.Build
+import androidx.annotation.RequiresApi
+
+object RapidPermissionHelper {
+
+    /**
+     * Get list of permissions required for read and write
+     * Since [Build.VERSION_CODES.TIRAMISU] need [Manifest.permission.READ_MEDIA_IMAGES], [Manifest.permission.READ_MEDIA_VIDEO], [Manifest.permission.READ_MEDIA_AUDIO]
+     * Below [Build.VERSION_CODES.TIRAMISU] need [Manifest.permission.WRITE_EXTERNAL_STORAGE], [Manifest.permission.READ_EXTERNAL_STORAGE]
+     */
+    fun requiredPermissionsForReadAndWrite(): Array<String> {
```

Comment:
Read and Write of what?? Could be file, SMS, calllog? Please specify clearly in the name